.github/workflows/release-please.yml

```diff
@@ -0,0 +1,22 @@
+name: release-please
```

Comment:
This will create release PRs based on conventional commits.

I kept the built in GitHub token until we install the GitHub app for the repo